core/model/src/main/java/org/eclipse/sw360/antenna/model/util/ArtifactLicenseUtils.java

```diff
@@ -54,13 +57,33 @@ public static LicenseInformation getFinalLicenses(Artifact artifact) {
         final Optional<LicenseInformation> declared = artifact.askForGet(DeclaredLicenseInformation.class);
         final Optional<LicenseInformation> observed = artifact.askForGet(ObservedLicenseInformation.class);
         if(declared.isPresent() && observed.isPresent()) {
-            final LicenseStatement effective = new LicenseStatement();
-            effective.setLeftStatement(declared.get());
-            effective.setRightStatement(observed.get());
-            effective.setOp(LicenseOperator.AND);
-            return effective;
-        }
+            List<License> allLicenses = new ArrayList<>(
+                    Stream.of(declared.get().getLicenses(), observed.get().getLicenses())
```

Comment:
This flattens the license list and removes all information about `AND` or `OR` conjunctions between licenses.